WARNING  viperlog:logger.py:109 Cannot expand rows in table SOURCE to ['SOURCE_ID', 'TIME', 'SPECTRAL_WINDOW_ID'], possibly duplicate values in those coordinates. Exception: unmatched dimensions for multi-index variables 'SOURCE_ID' ('SOURCE_ID',), 'TIME' ('TIME',), 'SPECTRAL_WINDOW_ID' ('SPECTRAL_WINDOW_ID',)